arangod/Replication2/ReplicatedLog/LogEntries.h

```diff
@@ -27,8 +27,11 @@
 #include <velocypack/Buffer.h>
 #include <velocypack/Slice.h>
 
+#include "Replication2/ReplicatedLog/AgencyLogSpecification.h"
 #include "Replication2/ReplicatedLog/LogCommon.h"
 
+using namespace arangodb::replication2::agency;
```

Comment:
`using` in a header is unfriendly, because it pollutes the global namespace of all includers.  😞 I know ... I would like to have that as well.